Governance Polls 2025-03-10 (#2860)
makerdao/community: governance/polls/Spark Liquidity Layer Arbitrum - Onboard Fluid sUSDS - March 10, 2025.md

---
title: Spark Liquidity Layer Arbitrum - Onboard Fluid sUSDS - March 10, 2025
summary: Signal your support or opposition to onboarding Fluid sUSDS onto the Spark Liquidity Layer on Arbitrum.
discussion_link: https://forum.sky.money/t/march-20-2025-proposed-changes-to-spark-for-upcoming-spell/26113
parameters:
    input_format: single-choice
    victory_conditions:
        - { type : plurality }
    result_display: single-vote-breakdown
version: v2.0.0
options:
   0: Abstain
   1: Yes
   2: No
start_date: 2025-03-10T16:00:00
end_date: 2025-03-13T16:00:00
---
# Poll: Spark Liquidity Layer Arbitrum - Onboard Fluid sUSDS - March 10, 2025

The Governance Facilitators have placed a Governance Poll into the voting system on behalf of the Stability Facilitators. This Governance [Poll](https://sky-atlas.powerhouse.io/#A.1.9.1_Operational_Weekly_Cycle-b189fa17-57a9-4d4e-9780-0ce4efd94211%7C0db30308) will be active for three days beginning on Monday, March 10 at 16:00 UTC.

**This is a binary vote.**

- **You may vote for a single option.**
- **You should vote for the option which you prefer.**
- **If you would accept either option, you should vote 'Abstain'.**

## Review

> The community can vote in this poll to express support or opposition to onboarding Fluid sUSDS onto the Spark Liquidity Layer on Arbitrum with the following parameters:

Reviewer: ```suggestion
The community can vote in this poll to express support or opposition to onboarding Fluid sUSDS onto the Spark Liquidity Layer on Arbitrum with the following [Rate Limits](https://docs.spark.fi/dev/spark-liquidity-layer/spark-alm-controller#rate-limits):
```

- Deposits
  - `max amount`: **10 million USDS worth of sUSDS**
  - `slope`: **5 million USDS worth of sUSDS**
- Withdrawals
  - `max amount`: **unlimited**

Please review the discussion [thread](https://forum.sky.money/t/march-20-2025-proposed-changes-to-spark-for-upcoming-spell/26113) to help inform your position before voting. Please refer to the [risk assessment post](https://forum.sky.money/t/instadapp-fluid-risk-assessment/25889) for details about Fluid.

## Outcomes

**If the votes for the 'Yes' option exceed the votes for the 'No' option then the following actions will be taken:**

- This change will be included in an upcoming Executive Vote.
- It is expected that this Executive Vote will take place within 30 days of this poll passing, absent external factors.

> - If the Executive Vote passes, then these changes will become active in the Maker Protocol after the [GSM Pause Delay](https://sky-atlas.powerhouse.io/#A.1.8.2.1_Pause_Delay-a98b8227-95f6-4711-9d8d-f52cbc6ad2d0%7C0db30758e055) has expired.

Reviewer: ```suggestion
- If the Executive Vote passes, then these changes will become active in the Sky Protocol after the [GSM Pause Delay](https://sky-atlas.powerhouse.io/#A.1.8.2.1_Pause_Delay-a98b8227-95f6-4711-9d8d-f52cbc6ad2d0%7C0db30758e055) has expired.
```

**If the votes for the 'No' option equal or exceed the votes for the 'Yes' option then no further action will be taken at this time.**

---

## Resources

> If you are new to voting in the Maker Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.

Reviewer: ```suggestion
If you are new to voting in the Sky Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.
```
